kairon/chat/handlers/channels/whatsapp.py

```diff
@@ -48,6 +48,8 @@ async def message(
             if message['type'] == "voice":
                 message['type'] = "audio"
             text = f"/k_multimedia_msg{{\"{message['type']}\": \"{message[message['type']]['id']}\"}}"
+        elif message.get("order"):
```

Comment:
add this slot as a kairon default slot, otherwise this will never keep the value at runtime

Reply:
create a order slot of any type